src/wallet/rpcdump.cpp

```diff
@@ -153,6 +154,91 @@ UniValue importprivkey(const UniValue& params, bool fHelp)
     return NullUniValue;
 }
 
+UniValue importprivatekeys(const UniValue& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+
+    if (fHelp || params.size() < 1)
+        throw runtime_error(
+            "importprivatekeys [rescan | no-rescan] \"bitcoinprivkey\"...\n"
+            "\nAdds private keys (as returned by dumpprivkey) to your wallet.\n"
+            "\nArguments:\n"
+            "1. \"rescan | no-rescan\" (string, optional default rescan) If \"no-rescan\", skip wallet rescan\n"
+            "2. \"bitcoinprivkey\"   (string, at least 1 required) The private keys (see dumpprivkey)\n"
+            "\nNote: This command will return before the rescan (may take hours) is complete.\n"
+            "\nExamples:\n"
+            "\nDump a private key\n"
+            + HelpExampleCli("dumpprivkey", "\"myaddress\"") +
+            "\nImport the private key with rescan\n"
+            + HelpExampleCli("importprivatekey", "\"mykey\"") +
+            "\nImport using a label and without rescan\n"
+            + HelpExampleCli("importprivkey", "no-rescan \"mykey\"") +
```

Comment:
shouldn't both examples should say "importprivatekeys"  and not "importprivatekey" or "importprivkey"